tests/crescent/utils/test_any_issubclass.py

```diff
@@ -0,0 +1,21 @@
+from crescent.utils import any_issubclass
+
+
+class A:
+    ...
+
+
+class B(A):
+    ...
+
+
+def test_any_issubclass_with_any():
+    assert not any_issubclass(None, B)
+
+
+def test_any_issubclass_with_cls():
+    assert any_issubclass(B, A)
+
+
+def test_any_issubclass_with_cls_false():
+    assert not any_issubclass(A, B)
```

Comment:
I would add a test on `Annotated[int, 'thing']` as well as `object()`

Reply:
Added a test for object, but imo testing `Annotated` doesn't make sense here. Its not really related to the point of the function.